src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -574,7 +574,7 @@ private void OnMouseLeftUseLevel(object parameter)
         /// </summary>
         protected virtual void RefreshPortColors()
         {
-            PortBackgroundColor = node.IsVisible ? PortBackgroundColorDefault : PortBackgroundColorPreviewOff;
+            PortBackgroundColor = PortBackgroundColorDefault;
```

Comment:
if the background is always being set to the same color just set it once.

Reply:
Discussed in person with @mjkkirschner. Leaving for now.